slices/vim-common.yaml

```
@@ -0,0 +1,13 @@
+package: vim-common
+
+essential:
+  - vim-common_copyright
+
+slices:
+  config:
+    contents:
+      /etc/vim/vimrc:
+
```

```suggestion
  addons:
    contents:
      /var/lib/vim/addons/:

```

We probably need to add this `addons`. Although it's empty, this might be used by other packages/slices while they assume the existence of this path is already established.